resolwe/flow/engines/celery.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sys
+
+from . import BaseManager
+from ..tasks import celery_run
+
+try:
```

Comment:
I see some issues with this:
* First, celery is already indirectly imported above when importing `..tasks`. So it will fail there.
* Second, you should not just print something on an `ImportError` and continue executing if the other code is not ready for this. Otherwise, the code could just fail later and the `print` output will simply be missed.